internal/service/team/update_user.go

```diff
@@ -0,0 +1,89 @@
+package team
```

Comment:
knit: not sure about naming the file specific for update users, instead of something more general

Reply:
dont have a strong opinion, I believe this is fairly particular case where we have a lot of business logic for handling the update of 1 attributes given it consumes multiple endpoints